packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -105,24 +104,22 @@ class FloatingActionButton extends StatelessWidget {
     this.foregroundColor,
     this.backgroundColor,
     this.heroTag = const _DefaultHeroTag(),
-    this.elevation = 6.0,
-    this.highlightElevation = 12.0,
-    double disabledElevation,
+    this.elevation,
+    this.highlightElevation,
+    this.disabledElevation,
     @required this.onPressed,
-    this.shape = const StadiumBorder(),
+    this.shape,
     this.isExtended = true,
     this.materialTapTargetSize,
     this.clipBehavior = Clip.none,
     Widget icon,
     @required Widget label,
-  }) : assert(elevation != null && elevation >= 0.0),
-       assert(highlightElevation != null && highlightElevation >= 0.0),
+  }) : assert(elevation == null || elevation >= 0.0),
```

Comment:
Same thing here, update the documentation above

Reply:
Done